projects/js-packages/publicize-components/src/components/twitter/options.js

```diff
@@ -26,20 +27,23 @@ export const PublicizeTwitterOptions = ( {
 	setTweetstorm,
 	prePublish,
 } ) => {
+	const tweetTypeChange = useCallback(
+		value => {
+			if ( 'tweetstorm' === value ) {
+				setTweetstorm( true );
+			} else {
+				setTweetstorm( false );
+			}
```

Comment:
This is purely me trying to find _something_ for this review, but this could be simplified to `setTweetstorm( 'tweetstorm' === value );`

Reply:
Good point, this was the original code. I think I'll tweak that in the next one.